cypress/elements/dimensionModal/dataDimension.js

```diff
@@ -78,6 +80,9 @@ export const switchDataTab = (tabName) => {
 export const expectDataTypeToBe = (type) =>
     cy.getBySel(dataTypesSelectButtonEl).should('contain', type)
 
+export const expectDataTypeSelectHelpToContain = (text) =>
+    cy.getBySel(dataTypesSelectHelpEl).should('contain', text)
```

Comment:
use `have.text` over `contain` for a tighter match

Reply:
Modified.